idea_inspection.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+
+#################################################
+# IntelliJ IDEA inspections for checkstyle.
+#################################################
+
+PROJECT_DIR=$PWD/
+INSPECTIONS_PATH=$PWD/config/intellij-idea-inspections.xml
+RESULTS_DIR=$PWD/target/inspection-results
+NOISE_LVL=v1
+IDEA_LOCATION=
+export IDEA_PROPERTIES=$PWD/config/idea.properties
+
+# Check IDEA_HOME env variable
+
+if [[ ! -z $IDEA_HOME ]]; then
```

Comment:
> 
```
$ ./idea_inspection.sh
bash: ./idea_inspection.sh: Permission denied
```

please do `chmod u+x ./idea_inspection.sh` and push to repo file that will be ready to use.

-------------------------
>
 ```
$ ./idea_inspection.sh
./idea_inspection.sh: 16: ./idea_inspection.sh: [[: not found
IDEA_HOME variable not found.
```
first error is not expected.

Reply:
done